webapp/components/team_general_tab.jsx

```diff
@@ -233,7 +237,7 @@ class GeneralTab extends React.Component {
         const {formatMessage} = this.props.intl;
         const description = this.state.description.trim();
         if (description === this.props.team.description) {
-            state.clientError = formatMessage(holders.chooseName);
+            state.clientError = formatMessage(holders.chooseDescription);
```

Comment:
Can you actually use `Utils.localizeMessage()` here? We're trying to get rid of uses of `formatMessage`